packages/core/src/api/tests/formatConversion/export/getExportTestCases.ts

```diff
@@ -0,0 +1,1521 @@
+import { PartialBlock } from "../../../../blocks/defaultBlocks.js";
+import {
+  TestBlockSchema,
+  TestInlineContentSchema,
+  TestStyleSchema,
+} from "../../testSchema.js";
+
+export type ExportTestCase<
+  // At some point we probably want to only have one HTML format that is both
+  // lossless when converting to/from conversionType, content, in which case we will only need
+  // "html" test cases and can remove "blockNoteHTML".
+  ConversionType extends "blocknoteHTML" | "html" | "markdown" | "nodes"
```

Comment:
I'm not sure the generic here and below adds anything. you could just say `converstionType: "blocknoteHTML" | "html" | "markdown" | "nodes"`, right?